pkg/eval/json_evaluator.go

```diff
@@ -0,0 +1,50 @@
+package eval
```

Comment:
The only current IEvaluator implementation. It maintains the flag state and knows how to intelligently parse it to evaluate flags as per OpenFeature spec.